instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/test_async_chat_completions.py

```diff
@@ -633,6 +633,55 @@ async def test_async_chat_completion_multiple_tools_streaming_no_content(
     )
 
 
+@pytest.mark.vcr()
+@pytest.mark.asyncio()
+async def test_async_chat_completion_streaming_sample_out(
```

Comment:
dismiss if "sampled out" is a common phrase

```suggestion
async def test_async_chat_completion_streaming_unsampled(
```